src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/TypeLoaderExports.cs

```diff
@@ -162,8 +163,20 @@ private static unsafe Value CacheMiss(IntPtr context, IntPtr signature, RuntimeO
     internal static unsafe class RawCalliHelper
     {
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static void Call(System.IntPtr pfn, ref byte data)
-            => ((delegate*<ref byte, void>)pfn)(ref data);
+        public static void CallDefaultStructConstructor(System.IntPtr pfn, ref byte data)
+        {
+            // Manually expand call of the instance method fat pointer. We cannot use a regular C# function
+            // pointer call because it cannot express call of an instance method.
```

Comment:
```suggestion
            // Manually expand call of the instance method fat pointer. We cannot use a regular static C# function
            // pointer call since it would not work for shared generic instance method.
```